Taxes/Operation/01_ManageDecisionMatrix.md

```diff
@@ -0,0 +1,256 @@
+[!!Accounting](RetailSuiteAccounting)  
+[!!Fakturierung](RetailSuiteFaktBase)  
+[!!User Interface Decision matrix](../UserInterface/01b_DecisionMatrix.md)  
+[!!Manage the tax rates](../Integration/01_ManageTaxRates.md)  
+[!!Manage the tax classes](../Integration/02_ManageTaxClasses.md)  
+[!!Manage the tax zones](../Integration/03_ManageTaxZones.md)  
+
+
+# Manage the decision matrix
+
+The decision matrix is designed to help the system decide which tax zone, and therefore which tax rate, is applicable to an order. The decision matrix consists of a list of decision-making criteria based on the information provided by the *Invoicing* module.
+
+Taking into account the order confirmation and the delivery note, the *Invoicing* module provides the *Taxes* module with the following details: country of origin, ship-to country, ship-to postcode, and VAT ID. Using this information, the decision matrix is able to determine the tax zone. Once the tax zone has been determined, the *Taxes* module can provide the *Invoicing* module with this information, as well as with the applicable tax key and tax rate.  
+
+The correct order of the decision-making criteria is crucial, as the system goes through the criteria in the order they are listed, starting from the top, and stops checking them as soon as a matching criterion is found. This means, that the criteria must be organized from specific to general to cover all possible relevant cases.
+
+
+## Create an entry
+
+Create an entry to define a new tax zone.
+
+#### Prerequisites  
+
+- An applicable tax class is created, see [Create a tax class](../Integration/02_ManageTaxClasses.md#create-a-tax-class).
+- An applicable tax zone is created, see [Create a tax zone](../Integration/03_ManageTaxZones.md#create-a-tax-zone).
+
+#### Procedures
+
+*Taxes > Settings > Tab DECISION MATRIX*
+
+![Decision matrix](../../Assets/Screenshots/Taxes/Settings/DecisionMatrix/DecisionMatrix.png "[Decision matrix]")
+
+1. Click the ![Add](../../Assets/Icons/Plus01.png "[Add]") (Add) button in the upper left corner.   
+  A new data entry line is displayed.
+
+2. Click the *VAT ID* drop-down list and select the appropriate option. The following options are available:
+
+  - **Yes**: The end customer is a company and, therefore, has a VAT ID number (business-to-business transaction).
+  - **No**: The end customer is a private individual and, therefore, has no VAT ID number (business-to-customer transaction).
+  - **Any**: Wildcard; any value is accepted.
```

Comment:
"Any" ist keine option in der drop-down liste, sondern wird nur dann angezeigt, wenn weder "Yes" noch "No" ausgewählt ist.